packages/elements/src/photon-multirx-form/components/PatientCard.tsx

```diff
@@ -50,15 +53,71 @@ export const PatientCard = (props: {
     }
   };
 
+  const fetchOrCreatePatient = async () => {
+    const { data } = await props.client!.getSDK().clinical.patient.getPatients({
+      name: `${props?.patient?.firstName} ${props?.patient?.lastName}`
+    });
```

Comment:
I thnk we should be able to do a call to the `patient` query and pass in external id